packages/vats/src/core/types-ambient.d.ts

```diff
@@ -342,6 +344,7 @@ type ChainBootstrapSpaceT = {
   client: ClientManager;
   clientCreator: any;
   coreEvalBridgeHandler: import('../types.js').BridgeHandler;
+  cosmosInterchainService: import('@agoric/orchestration/src/exos/cosmos-interchain-service.js').CosmosInterchainService;
```

Comment:
This was once available via `index.js` - i think we missed a rename of `export * from './src/service.js';`

Reply:
nope it's still available there. This is just what autocompleted. I'll simplify it since I have to squash fixups anyway